itou/users/forms.py

```diff
@@ -1,6 +1,17 @@
 from django.core.exceptions import ValidationError
 
 from itou.users.models import JobSeekerProfile
+from itou.utils import constants as global_constants
+
+
+def validate_francetravail_email(email):
+    allowed_suffixes = [
+        global_constants.POLE_EMPLOI_EMAIL_SUFFIX,
+        global_constants.FRANCE_TRAVAIL_EMAIL_SUFFIX,
+    ]
+    if not any(email.endswith(s) for s in allowed_suffixes):
```

Comment:
Je passais là par curiosité mais je suis tombé sur un truc marrant pour `.startswith()` et `.endswith()` :
> [...] suffix can also be a tuple of suffixes to look for. [...]
> https://docs.python.org/3/library/stdtypes.html#str.endswith

```suggestion
    allowed_suffixes = (
        global_constants.POLE_EMPLOI_EMAIL_SUFFIX,
        global_constants.FRANCE_TRAVAIL_EMAIL_SUFFIX,
    )
    if not email.endswith(allowed_suffixes):
```